README.md

```diff
@@ -133,6 +133,7 @@ Events emitted by the component to the parent.
 | vdropzone-file-added(file) | File added to the dropzone.|
 | vdropzone-files-added(file) | Files added to the dropzone.|
 | vdropzone-file-added-manually(file) | Manually added file to the dropzone |
+| vdropzone-thumbnail(file, dataUrl) | When the thumbnail has been generated. Receives the dataUrl as second parameter. |
```

Comment:
`dataUrl` will be **base64 encoded** string right? 

Do you think it's worth to mention in docs about it like below? 

 `Receives the base64 encoded dataUrl as second parameter.`

Reply:
No because the dataUrl can also be a URL, when you add a file manually, its only a base64 when you drag files into it :)

Reply:
By `when you add a file manually` I'm talking about the method `manuallyAddFile`

Reply:
Great!